src/main/java/de/ellpeck/rockbottom/world/tile/TileWoodDoor.java

```diff
@@ -96,6 +99,13 @@ public boolean isFullTile() {
         return false;
     }
 
+    @Override
+    public boolean canLiquidSpread(IWorld world, int x, int y, TileLiquid liquid, Direction dir) {
+        TileState state = world.getState(x, y);
+        boolean facingRight = state.get(StaticTileProps.FACING_RIGHT);
+        return state.get(StaticTileProps.OPEN) || facingRight && dir == Direction.LEFT || !facingRight && dir == Direction.RIGHT;
```

Comment:
i think we are doing `return state.get(StaticTileProps.OPEN) || (facingRight && dir == Direction.LEFT) || (!facingRight && dir == Direction.RIGHT);`

Reply:
Yes that is what we are doing

Reply:
But that's not what you did

Reply:
what? Yes it is.